docs/document/docs/components/desktop/dialog.md

```diff
@@ -25,6 +25,7 @@ Here is a list of properties that can be used for modifying the component:
 
 | Name | Type | Default | Description | Remark |
 | :--- | :--- | :--- | :--- | :--- |
+| icon | string | "" | The icon in content area | Valid value:<li>"info" : ![info](assets/icon-info.png)</li><li>"success" : ![success](assets/icon-success.png)</li><li>"error" : ![error](assets/icon-error.png)</li><li>"warning" : ![warning](assets/icon-warning.png)</li><li>"question" : ![question](assets/icon-question.png)</li><li>"" : No icon</li> |
```

Comment:
I think it would be better to match the similar description of other components.
What do you think?

```suggestion
| icon | string | "" | The icon in content area | Available options:<li>"info" : ![info](assets/icon-info.png)</li><li>"success" : ![success](assets/icon-success.png)</li><li>"error" : ![error](assets/icon-error.png)</li><li>"warning" : ![warning](assets/icon-warning.png)</li><li>"question" : ![question](assets/icon-question.png)</li><li>"" : No icon</li> |
```

ref: https://github.com/kintone-labs/kintone-ui-component/blob/v1.4.0_SSR-2336_document-Dialog-icon-property/docs/document/docs/components/desktop/button.md?plain=1#L29